aries-site/src/pages/tokens/overview.mdx

```diff
@@ -0,0 +1,80 @@
+import { Example } from '../../layouts';
+import { Box, Text } from 'grommet';
+
+## What is a design token?
+
+Design tokens codify the design decisions that make up a design system's visual style. Tokens replace hard-coded values, such as hex codes for color or px values for spacing, with clear names.
```

Comment:
I think we should try call them design tokens as much as possible. It feels wordy, but given that token has other meanings in the engineering world, would be a good idea.

Reply:
Ok cool, in that case I would review all the text and change tokens for design tokens, specially on the title of this branch of documentation.

Reply:
@taysea I saw you closed this but we are still calling this whole section "Tokens" and the documentation still uses both terms interchangeably. If you guys think that's ok I am good with it but wanted to confirm since @oliverHPE's comment made me understand differently.

Reply:
Ah thank you, I'll update.

Reply:
Have updated generic use of word "tokens" to "design tokens"